samples/ListFonts/Program.cs

```diff
@@ -7,25 +7,30 @@
 using System.Linq;
 using System.Threading;
 using SixLabors.Fonts;
-using SixLabors.Fonts.Unicode;
 
 namespace ListFonts
 {
     public static class Program
     {
         public static void Main(string[] args)
         {
-            IEnumerable<FontFamily> families = SystemFonts.Families;
-            IOrderedEnumerable<FontFamily> orderd = families.OrderBy(x => x.Name);
-            int len = families.Max(x => x.Name.Length);
-            foreach (FontFamily f in orderd)
+            var pairings = new List<Pairing>();
+            IOrderedEnumerable<FontFamily> ordered = SystemFonts.Families.OrderBy(x => x.Name);
+            foreach (FontFamily family in ordered)
             {
-                Console.Write(f.Name.PadRight(len));
-                Console.Write('\t');
-                Console.Write(string.Join(",", f.AvailableStyles.OrderBy(x => x).Select(x => x.ToString())));
-                Console.WriteLine();
+                IOrderedEnumerable<FontStyle> styles = family.GetAvailableStyles().OrderBy(x => x);
+                foreach (FontStyle style in styles)
+                {
+                    Font font = family.CreateFont(0F, style);
+                    font.TryGetPath(out string path);
+                    pairings.Add(new Pairing(font.Name, path));
```

Comment:
a simple named tuple was probably all you needed here instead of creating the full struct, but not particually important and isn't really an issue I would flag up to change.

Reply:
Yeah, agree, that would have been better. Tbh I used  a struct because I always forget the syntax and it was quicker than searching!